pysb/simulator/base.py

```diff
@@ -322,12 +474,12 @@ def __init__(self, simulator, trajectories):
                 raise ValueError("trajectories should be a 3D array or a list "
                                  "of 2D arrays")
             self._y = trajectories
-
-        self.nsims = len(self._y)
-        if len(self.tout) != self.nsims:
+        
+        self._nsims = len(self._y)
```

Comment:
I think it could be useful to keep this public, but read-only. Can we have a getter `nsims` that returns `self._nsims`? It would clean up a lot of the simulator code too, to revert all uses of `self._nsims` to `self.nsims`.